src/ImageWrapper.cpp

```diff
@@ -3,16 +3,85 @@
 extern "C" {
   #include <Gosu/Image.h>
 
+  // Constructors
   Gosu_Image* Gosu_Image_create(char* filename, unsigned image_flags)
   {
     return reinterpret_cast<Gosu_Image*>( new Gosu::Image(std::string(filename), image_flags) );
   };
 
+  // Properties
+  int Gosu_Image_width(Gosu_Image* image)
+  {
+    return reinterpret_cast<Gosu::Image*>( image )->width();
+  }
+
+  int Gosu_Image_height(Gosu_Image* image)
+  {
+    return reinterpret_cast<Gosu::Image*>( image )->height();
+  }
+
+  // Rendering
+  void Gosu_Image_draw(Gosu_Image* image, double x, double y, double z, unsigned color, int mode)
+  {
+    reinterpret_cast<Gosu::Image*>( image )->draw(x, y, z, 1.0, 1.0, color, (Gosu::AlphaMode)mode);
+  }
+
+  void Gosu_Image_draw_rot(Gosu_Image* image, double x, double y, double z,
+                           double angle, double center_x, double center_y,
+                           double scale_x, double scale_y, unsigned color, int mode)
+  {
+    reinterpret_cast<Gosu::Image*>( image )->draw_rot(x, y, z,
```

Comment:
Do you think it would be worth the CPU cycles to check if `image != nullptr`? Although you couldn't really do anything useful but `abort` anyway.